bin/-tarball-info.js

```diff
@@ -77,7 +77,7 @@ const path = require('path');
 
 const projectRoot = path.resolve(__dirname, '../');
 // we share this for the build
-const tarballDir = path.join(projectRoot, '../__tarball-cache');
+const tarballDir = '../../__tarball-cache';
```

Comment:
Not sure if windows support matters but I think we can also use `path.relative(__dirname, path.join(projectRoot, '../__tarball-cache'));` to get a platform friendly path. 

Unfortunately, published packages wont work on windows since a unix path would likely be generated a build time and encoded into the published `package.json` tarballs, but thats an issue for the future since these tarballs arn't published anywhere yet.